DeviceDetector.php

```diff
@@ -774,7 +774,7 @@ protected function getOsAttribute(string $attr): string
      *
      * @return bool
      */
-    protected function hasAndroidTableFragment(): bool
+    protected function hasAndroidTabletFragment(): bool
```

Comment:
To avoid mistakes in projects, who could use this method.
I suggest safe the old name `hasAndroidTableFragment`